packages/migrate/src/__tests__/MigrateNew.test.ts

```diff
@@ -0,0 +1,348 @@
+import { jestConsoleContext, jestContext } from '@prisma/get-platform'
+import fs from 'fs-jetpack'
+
+import { MigrateNew } from '../commands/MigrateNew'
+import { CaptureStdout } from '../utils/captureStdout'
+import { setupMysql, tearDownMysql } from '../utils/setupMysql'
+import type { SetupParams } from '../utils/setupPostgres'
+import { setupPostgres, tearDownPostgres } from '../utils/setupPostgres'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+const captureStdout = new CaptureStdout()
+
+// Disable prompts
+process.env.GITHUB_ACTIONS = '1'
+// Disable generate
+process.env.PRISMA_MIGRATE_SKIP_GENERATE = '1'
+
+const originalEnv = { ...process.env }
+
+beforeEach(() => {
+  captureStdout.startCapture()
+})
+
+afterEach(() => {
+  captureStdout.clearCaptureText()
+})
+
+afterAll(() => {
+  captureStdout.stopCapture()
+})
+
+describe('common', () => {
+  it('invalid schema', async () => {
```

Comment:
Should this actually happen if you just want to create an _empty_ SQL migration file independent from your Prisma Schema?